src/libasr/codegen/asr_to_wasm.cpp

```diff
@@ -61,6 +61,20 @@ struct SymbolFuncInfo {
     Vec<ASR::Variable_t *> referenced_vars;
 };
 
+enum RT_FUNCS {
+    print_i64 = 0,
+    print_f64 = 1,
+    add_c32 = 2,
+    add_c64 = 3,
+    sub_c32 = 4,
+    sub_c64 = 5,
+    mul_c32 = 6,
+    mul_c64 = 7,
+    abs_c32 = 9,
+    abs_c64 = 10,
+};
+const int NO_OF_RT_FUNCS = 11;
```

Comment:
This might be more maintainable:
```suggestion
    rt_funcs_last = 11, // keep this as the last enumerator
};
const int NO_OF_RT_FUNCS = rt_funcs_last;
```